demos/default/pages/broken-image.tsx

```diff
@@ -1,4 +1,4 @@
-import Image from 'next/legacy/image'
+import Image from 'next/image'
 
 // This should cause an error, because broken-domain is not part of the configured next.config.js image domains
 const Images = () => (
```

Comment:
[pebble] I feel this test would be more robust if the image URL pointed to a real image that's not on the allowlist

Reply:
Done